ntbs-service/Services/UserService.cs

```diff
@@ -5,67 +5,87 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Options;
 using ntbs_service.Models;
+using ntbs_service.Models.Enums;
```

Comment:
quite a bit of refactoring here, as rather than just returning tb services we need to know tbservice or phecs that are viewable by a particular user, so I created a new "UserPermissionsFilter" model to encompass both. If you can think of a snappier name for that let me know